android/src/main/java/com/paywallsdkreactnative/HeliumBridge.kt

```
@@ -424,8 +542,27 @@ class HeliumBridge(private val reactContext: ReactApplicationContext) :
+        Helium.config.logger = HeliumLogger.Stdout
+        BridgeStateManager.clearPendingEvents()
+        try {
+            Helium.resetHelium(
+                clearUserTraits = clearUserTraits,
+                clearHeliumEventListeners = clearHeliumEventListeners,
+                clearExperimentAllocations = clearExperimentAllocations,
+                onComplete = {
+                    promise.resolve(null)
+                }
+            )
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)